packages/dispatcher-converter/src/util/FileOperations.js

```diff
@@ -1432,8 +1432,6 @@ class FileOperations {
                                         "FileOperationsUtility: Commenting non-whitelisted directive usage in ",
                                         filePathWithLine
                                     );
-                                } else {
```

Comment:
These blocks are already covered in condition at https://github.com/adobe/aem-cloud-service-source-migration/blob/dd33b534c144fff187ee9718dd9ae50d77ecdcd2/packages/dispatcher-converter/src/util/FileOperations.js#L1489 . Keeping them are adding the lines twice hence the removal.